frontend/__tests__/components/cluster-settings.spec.tsx

```diff
@@ -233,6 +241,37 @@ describe('Cluster Version Details Table page while ClusterVersion Upgradeable=Fa
   });
 });
 
+describe('ClusterSettingsAlerts while cluster is up to date or has available updates and there are paused MachineConfigPools', () => {
+  let wrapper: ShallowWrapper<any>;
```

Comment:
Small nit. `React.ComponentProps<typeof Component>` is how you get props, instead of `any`.